app/models/vote.rb

```diff
@@ -0,0 +1,7 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  VOTES = ["up", "down"].freeze
+  validates :vote_type, presence: true, inclusion: { in: VOTES }
```

Comment:
Addressed in this commit: https://github.com/mcon85/doors_at_eight/pull/28/commits/76622824d55f469fb3e2a2662354f533eb27bb8a